src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -31,13 +30,11 @@ public class LineAcceptanceTest {
 	@Test
 	void createLine() {
 		// when
-		LineRequest lineRequest = new LineRequest("신분당선", "bg-red-600", 1L, 2L, 10);
-		createLine(lineRequest);
+		long id = lineApi.createLine("신분당선", "bg-red-600", 1L, 2L, 10).jsonPath().getLong("id");
 
 		// then
-		ExtractableResponse<Response> showResponse = showLines();
-		List<String> lineNames = showResponse.jsonPath().getList("name", String.class);
-		assertThat(lineNames).containsAnyOf("신분당선");
+		ExtractableResponse<Response> showResponse = lineApi.showLineById(id);
+		assertThat(showResponse.jsonPath().getString("name")).isEqualTo("신분당선");
```

Comment:
```suggestion
		String lineName = lineApi.showLineById(id);
		assertThat(lineName).isEqualTo("신분당선");
```

lineName을 주도록 변경하면 더 간결해지지 않을까요?